.gitignore

```diff
@@ -25,3 +25,8 @@ npm-debug.log
 # WebStorm user-specific
 .idea/workspace.xml
 .idea/tasks.xml
+.idea/cesium.iml
+.idea/encodings.xml
+.idea/inspectionProfiles/Project_Default.xml
+.idea/jsLinters/jshint.xml
+.idea/misc.xml
```

Comment:
I don't think we want to ignore any of these files (and they are also already committed in master, so ignoring them would product odd behavior).  What is your reasoning for adding them here?